pilot/ProfileViewController.swift

```diff
@@ -2,18 +2,36 @@
 //  ProfileViewController.swift
 //  pilot
 //
-//  Created by Nick Eckert on 9/26/17.
+//  Created by Nick Eckert on 9/29/17.
 //  Copyright © 2017 sanction. All rights reserved.
 //
 
 import UIKit
 
-class ProfileViewController: UIViewController {
-
+class ProfileViewController: UITableViewController {
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        navigationItem.largeTitleDisplayMode = .never
+        self.navigationItem.largeTitleDisplayMode = .never
+        
+        let customHeaderView = Bundle.main.loadNibNamed("CustomTableHeaderView", owner: self, options: nil)?.first as! UIView
+        self.tableView.tableHeaderView = customHeaderView
     }
-
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        if let headerView = tableView.tableHeaderView {
+            let height = headerView.systemLayoutSizeFitting(UILayoutFittingCompressedSize).height
+            var headerFrame = headerView.frame
+            
+            if height != headerFrame.size.height {
+                headerFrame.size.height = height
+                headerView.frame = headerFrame
+                tableView.tableHeaderView = headerView
```

Comment:
Do you need to do this reassignment here? When you change the `headerView.frame` in the line above, does it not change the `tableView.tableHeaderView` too?

Reply:
I changed this at a later date and turns out I don't need that!